stdlib/array.gr

```diff
@@ -638,6 +703,9 @@ provide let findIndex = (fn, array) => {
  * @param array1: The array to provide values for the first tuple element
  * @param array2: The array to provide values for the second tuple element
  * @returns The new array containing all pairs of `(a, b)`
+ * 
+ * @example
+ * Array.product([> 1, 2], [> 11, 12]) // [> (1, 11), (1, 12), (2, 11), (2, 12)]
```

Comment:
Same As Above:
```suggestion
 * Array.product([> 1, 2], [> 11, 12]) == [> (1, 11), (1, 12), (2, 11), (2, 12)]
```

Reply:
Same comment as before.  This is not returning void so simply in line the result.....